storybook/storybook-react/src/Link/Link.stories.tsx

```diff
@@ -68,23 +67,20 @@ const meta = {
   args: {
     children: 'Linktekst',
```

Comment:
This text has no descenders, unfortunately – they would help indicate the underline behaviour.

```suggestion
    children: 'Klik op mij',
```

We’ll need to think of something English in a few days or so.

Reply:
Is this still necessary with the 6px offset?

Reply:
I’m still not fond of ‘Linktekst’, let’s revisit when translating.